INSTALL.md

```diff
@@ -1,11 +1,15 @@
 Installing From Sources
 =======================
 
-This document presents instructions to install this branch of Coq.
-For more general installation instructions and information about known
-build system issues, please consult the wiki page: 
+If you are just a user, you should probably install Coq using the
+platform or a package manager (e.g. opam or Nix) instead.
 
-  https://github.com/coq/coq/wiki#coq-installation
+See https://coq.inria.fr/download and
+https://github.com/coq/coq/wiki#coq-installation to learn more.
+
+If you are a developer or you want to package Coq or you really need
+to build Coq from sources without a package manager like opam or Nix,
+then read on.
```

Comment:
The "technical" below is intended to express the "discourage" that you want to express:
```suggestion
Otherwise, the remainder of this file gives technical instructions for
building Coq from sources.
```